### PR TITLE
Added Contributor title to each contributor and added PA title to @anmode also added the no. of commits

### PR DESCRIPTION
**Pull request by Sch476**
## Related Issue

Closes: #698 

## Description of Changes

Added Contributor title to each contributor and added PA title to @anmode  also added the no. of commits and also made it clickable .

## Checklist:

- [ x] My code adheres to the established style guidelines of this project.
- [x ] I have conducted a self-review of my code.
- [ x] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [ ] My changes have not introduced any new warnings.

## Screencast

https://github.com/anmode/grabtern-frontend/assets/127319225/b8fdcef5-70de-415c-9509-ffc9df1d1a7c

### CODE REVIEW

File: components/Community/CommunityCard.jsx [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
 import { ProfileCard } from "../UI";
 
-const CommunityCard = ({ image, name, description}) => {
+const CommunityCard = ({ image, name, description, commitCount }) => {
+  // Custom label for the username "anmode"
+  const contributorLabel = name.toLowerCase() === "anmode" ? "Project Admin" : "Contributor";
+
+  // Function to generate the URL for the GitHub commits page for the contributor
+  const getCommitsURL = (username) => {
+    return `https://github.com/anmode/grabtern-frontend/commits?author=${username}`;
```
Comment (anmode) [inline components/Community/CommunityCard.jsx:10]: it will give me how many commits? isn't?

File: pages/community.page.js [hunk 2]
```diff
@@ -22,6 +21,40 @@ function Community() {
       });
   }, []);
 
+  // Function to fetch commit count for each contributor
+  const fetchCommitCount = (login) => {
+    return axios
+      .get(`https://api.github.com/repos/anmode/grabtern-frontend/commits`, {
+        params: {
+          author: login,
+        },
```
Comment (anmode) [inline pages/community.page.js:30]: commit counts are not correct!

**Review by anmode — CHANGES_REQUESTED** [review]

File: components/Community/CommunityCard.jsx (hunk 1 above)
Comment (Sch476) [reply components/Community/CommunityCard.jsx:10]: Yes

File: pages/community.page.js (hunk 2 above)
Comment (Sch476) [reply pages/community.page.js:30]: I understand what you are try to say. I am working on fixing it. The issue is if no. of commit > 30 it is showing 30. I get it.

**Comment by Sch476** [comment]
@anmode made the changes . Please check it.

**Comment by Sch476** [comment]
@anmode can I make a new PR for this ??
 
 
 Where I will implement the code freshly

**Comment by anmode** [comment]
Same again, 30 commit problem

**Pull request closed without merging**